packages/logger/src/logger.ts

```diff
@@ -42,7 +42,14 @@ export enum LogLevel {
 /**
  * The default log level
  */
-const defaultLogLevel: LogLevel = LogLevel.INFO;
+let defaultLogLevel: LogLevel = LogLevel.INFO;
```

Comment:
Default log level is a constant and should not be changed

Reply:
Resolving to consolidate similiar threads.